.github/workflows/build-images.yml

```diff
@@ -135,7 +134,7 @@ jobs:
           if [[ "${{ github.event_name }}" == 'schedule' ]]; then
               echo "::set-output name=cacheDirective::disabled"
           else
-              echo "::set-output name=cacheDirective::pulled"
+              echo "::set-output name=cacheDirective:registry"
```

Comment:
The "registry" is better name for the default caching as we are not pulling the image any more to use it as cache.